src/transformers/models/wav2vec2/modeling_flax_wav2vec2.py

```diff
@@ -840,7 +909,9 @@ def __call__(
             rngs=rngs,
         )
 
-    def _get_feat_extract_output_lengths(self, input_lengths: Union[jnp.ndarray, int]):
+    def _get_feat_extract_output_lengths(
```

Comment:
This will only be needed once we add `FlaxEncoderDecoder`. Let's revert this change for now as well ;-)